Generals/Code/GameEngine/Source/Common/GlobalData.cpp

```diff
@@ -37,6 +37,7 @@
 #define DEFINE_WEATHER_NAMES
 #define DEFINE_BODYDAMAGETYPE_NAMES
 #define DEFINE_PANNING_NAMES
+#define GAME_COMPAT_EXE_CRC 4241652750
```

Comment:
I suggest write this number as hex, because that is probably how it shows up in other views, for example `DEBUG_LOG(("EXE CRC: 0x%8.8X\n", m_exeCRC));`